app/tests/sticky-keys/11-lazy-timeout-during/native_posix_64.keymap

```diff
@@ -0,0 +1,37 @@
+#include <dt-bindings/zmk/keys.h>
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/kscan_mock.h>
+
+/* this test ensures that timing out while the other key is being held results in correct behavior */
+
+&sk {
+    lazy;
+    release-after-ms = <50>;
+};
+
+/ {
+    keymap {
+        compatible = "zmk,keymap";
+        label ="Default keymap";
```

Comment:
I read somewhere about labels being removed... just wanted to bring this to your attention if that's something you don't want to merge in.

Reply:
Yep. The label property is now deprecated. It won't hurt to include it (aside from a compiler warning that will probably just get ignored), but ideally it should be removed.